export/export_pipeline/compute_collision_representation.py

```diff
@@ -0,0 +1,427 @@
+                dst_filename = src_filename.replace(obj_name, '')
+                if dst_filename == '.stl':
+                    continue
+                shutil.move(osp.join(decompose_dir, src_filename), osp.join(decompose_dir, dst_filename))
+        elif acd_args['decompose_method'] == 'none':
+            out_filename = osp.join(decompose_dir, 'assembled.stl')
+            mesh.export(out_filename)
+            out_filename = osp.join(decompose_dir, '000.stl')
+            if params['common']['CONVEX_DECOMPOSITION_EXT'] in mesh_path:
+                shutil.copy(mesh_path, out_filename)
+            else:
+                mesh.export(out_filename)
+            decomposed = True
+        elif acd_args['decompose_method'] == 'skip':
+            decomposed = False
+        else:
+            raise ValueError(f'{mesh_dir} decompose method {acd_args["decompose_method"]} is invalid')
+        
+        print(f"Removing {mesh_path}")
+        os.remove(mesh_path)
+        
+    if not decomposed:  # delete whole directory
+        shutil.rmtree(mesh_dir)
+    return decomposed
+
+
+class CollisionRepresentationComputer(object):
+    def __init__(self, scene_path: str, n_workers: int, rerun: bool, include_objects: str) -> None:
+        self.n_workers = min(max(1, n_workers), mp.cpu_count()-1)
+        self.rerun = rerun
+        self.scene_path = osp.normpath(scene_path)
+        if include_objects is not None:
+            include_objects = include_objects.split(',')
+        self.input_gltf_filenames = []
+        with open(osp.join('export_pipeline', 'params.yaml'), 'r') as f:
+            self.p = yaml.safe_load(f)
+        self.input_dir = osp.join(scene_path, self.p['common']['GLTF_SCENE_DIR_NAME'])
+        self.output_dir = osp.join(scene_path, self.p['common']['COLLISION_DIR_NAME'])
+        for filename in next(os.walk(self.input_dir))[2]:
+            if not filename.endswith('.gltf'):
+                continue
+            if include_objects and not any([osp.splitext(filename)[0] == i for i in include_objects]):
+                continue
+            self.input_gltf_filenames.append(osp.join(self.input_dir, filename))
+
+
+    def run(self):
+        if not self.rerun:
+            shutil.rmtree(self.output_dir, ignore_errors=True)
+
+        bpy.ops.object.select_all(action='SELECT')
+        bpy.ops.object.delete()
+        
+        with open(osp.join(self.input_dir, 'actors_information.json'), 'r') as f:
+            actors_information = json.load(f)
+        
+        process_mesh_args = []
+        for input_gltf_filename in self.input_gltf_filenames:
+            bpy.ops.import_scene.gltf(filepath=input_gltf_filename)
+            obj = bpy.context.object
+            if obj.parent is not None:
+                continue
+            
+            object_name = obj.name.split('.')[0]
+            obj_dir = os.path.join(self.output_dir, object_name)
+            obj_info = actors_information[object_name]
```

Comment:
- The variable names in this section are a bit too terse, and should be more more consistent:

```
# less consistent
obj_info = actors_information[object_name]

# more consistent
actor_desc = actor_descs[actor_name]
```